widget/check_group.go

```diff
@@ -44,6 +46,19 @@ func (r *CheckGroup) Append(option string) {
 	r.Refresh()
 }
 
+// Remove removes the first specified option found from a CheckGroup widget.
```

Comment:
I think this wording is a bit unclear. It sounds almost as if it just removes the first item in the slice.

Reply:
Is this one better? [76ab4fb](https://github.com/fyne-io/fyne/pull/3323/commits/76ab4fb557ace3f4d21502c0246baa296217a8fd)

Reply:
We can also add remove on index, but it means adding an error. And since Options is public I am not sure of the value added.

Reply:
Let us know if this is fixed @Jacalz so it can land